kivy/core/camera/camera_opencv.py

```diff
@@ -138,9 +138,11 @@ def _update(self, dt):
             try:
                 self._buffer = frame.imageData
             except AttributeError:
-                # On OSX there is no imageData attribute but a tostring()
+                # On OSX/Linux there is no imageData attribute but a tostring()
                 # method.
-                self._buffer = frame.tostring()
+                # frame is already of type ndarray which can be reshaped to 1-d.
+                # Either frame.tostring() or frame.tobytes() is slow.
+                self._buffer = frame.reshape(-1)
```

Comment:
If the comments about tostring/tobytes are not relevant anymore, just remove them, and instead document why reshape does what we want (which maybe is what 143 is about).

Reply:
All the three choices tostring/tobytes/reshape do what you what but reshape should be chosen over the other two for efficiency. It is relevant for people unfamiliar with Numpy but irrelevant otherwise. There might be a reason why tostring() is used initially, which is unfortunately not explicitly commented though.